src/Symfony/Component/Translation/Translator.php

```diff
@@ -76,11 +70,15 @@ class Translator implements TranslatorInterface, TranslatorBagInterface
      *
      * @api
      */
-    public function __construct($locale, MessageSelector $selector = null, $cacheDir = null, $debug = false)
+    public function __construct($locale, MessageSelector $selector = null, $cache = null, $debug = false)
```

Comment:
The docblock needs to be changed to take into account that `$cache` can be an instance of `MessageCacheInterface`.